src/python/espressomd/magnetostatics.pyx

```diff
@@ -183,6 +183,10 @@ IF DP3M == 1:
 
             self._set_params_in_es_core()
             mpi_bcast_coulomb_params()
+        
+        def _deactivate_method(self):
+            dp3m_deactivate()
```

Comment:
Please check that you don't have to call `mpi_bcast_coulomb_params()` afterwards.

Reply:
That's already in the base call of `_deactivate_method` afterwards.